### DISCUSSION TITLE
Project related labels

### DISCUSSION THREAD

**Discussion by ebrarkiziloglu** [bounswe/bounswe2024group5#65] [Ideas]
Some label ideas for technical subsections of the project:
- backend
- front-end
- testing
etc.

**Comment by SouthAscend** [comment]
Since we're in the planning stage, I thought that we could put every label related to implementation in the next course, but if needed, we can add them now too.

**Reply by ebrarkiziloglu** [reply]
Yes I think that's reasonable. There was a discussion in the previous PS about this, that's why I came up with the idea. Yet, we will not need such labels for a long while.
